ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1936,6 +1936,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-automated-release
     workflow: cucushift-installer-rehearse-vsphere-upi-zones
+- as: openstack-ipi-f28-disasterrecovery
```

Comment:
4.13 reached EOL. Most of the jobs there are running with `f60`. It's better to align the test with the others. Of course, feel free to customize if you have a specific test requirements.

Reply:
Updated with `f60`.